doc/stdlib/linalg.rst

```diff
@@ -101,6 +101,10 @@ Linear algebra functions in Julia are largely implemented by calling functions f
 
    Compute a factorization of a positive definite matrix ``A`` such that ``A=L*Diagonal(d)*L'`` where ``L`` is a unit lower triangular matrix and ``d`` is a vector with non-negative elements.
 
+.. function:: ldltfact(A) -> CHOLMOD.Factor
+
+   Compute the LDLt factorization of a sparse symmetric or Hermitian matrix ``A``. A fill-reducing permutation is used.  The main application of this type is to solve systems of equations with ``\``, but also the methods ``diag``, ``det``, ``logdet`` are defined. The function calls the C library CHOLMOD and many other functions from the library are wrapped but not exported.
```

Comment:
I think there is two different issues here, but correct me if I'm wrong.

First of all, there is a bug in `ldltfact(SparseMatrixCSC)` which I'll have to look into. It appears that when CHOLMOD decides that a supernodal factorization is preferred, it switches to LLt even though I've tried to ask for the LDLt version. This doesn't happen for smaller matrices where the simplicial method is always used. This is probably why the tests didn't capture this.

I believe the second thing is mainly a naming convention and for what it is worth G&VL appears to use the same definition of LDLt as we do. As you know, we use `bkfact` for the pivoted version in the dense case. Even though you cannot be sure that the LDLt exists in general for Hermitian matrices, it is possible to LDLt factorize Hermitian matrices in many cases. Even indefinite matrices. Off course, if the decomposition fails, the function should give a reasonable error.